Doc/cmus.txt

```diff
@@ -499,6 +495,10 @@ mark <filter-expression>
 mute
 	Toggles mute for the sound output.
 
+passwd <password>
+	Set the password for TCP/IP connections. Required for connections if
+	bound to an IP address (see `--listen`).
+
```

Comment:
Hm, I have never used TCP/IP connections for cmus-remote, however testing it I found that I need to do the following:
```lang=bash
cmus --listen localhost:9999
:set passwd=asdfqwer
```
And for the remote session:
```lang=bash
cmus-remote --server localhost:9999 --passwd asdfqwer
```

Cmus does not recognise passwd as a command. cmus-remote does not complain, but I think it is only used internaly during the initial session setup.

Unless I missed something, I'd say that the new documentation is not correct either

Reply:
I'll look into it more when I have a chance (I've never used it either).